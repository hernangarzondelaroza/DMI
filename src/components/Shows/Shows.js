import React, { useEffect, useState } from "react";

import Show from "./Show/Show";
import ShowsService from "../../services/shows.service";

function Shows(props) {
  const [shows, setShows] = useState([]);
  // ComponentDidMount
  useEffect(() => {
    ShowsService.getShows()
      .then((response) => {
        setShows(response);
      })
      .catch((error) => console.log("Error", error.message));
  }, []);

  return <>{shows.data ? <Show data={shows.data} /> : ""}</>;
}

export default Shows;
