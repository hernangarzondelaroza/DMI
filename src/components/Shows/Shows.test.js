import React from 'react';
import renderer from 'react-test-renderer';
import Shows from './Shows';

describe('Shows', () => {
  it('Should take a snapshot', () => {
    const component = renderer
      .create(<Shows />);
    expect(component).toMatchSnapshot();
  })
})