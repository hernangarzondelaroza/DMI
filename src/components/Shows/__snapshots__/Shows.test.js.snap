// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shows Should take a snapshot 1`] = `""`;
